lms/djangoapps/teams/views.py

```diff
@@ -889,6 +888,13 @@ def post(self, request):
         except User.DoesNotExist:
             return Response(status=status.HTTP_404_NOT_FOUND)
 
+        course_module = modulestore().get_course(team.course_id)
+        if course_module.teams_max_size and team.users.count() >= course_module.teams_max_size:
+            return Response(
```

Comment:
Since you are using course_module.teams_max_size, you don't need to check if it exists. It always will.

    @property
    def teams_max_size(self):
        """
        Returns the max size for teams if teams has been configured, else None.
        """
        return self.teams_configuration.get('max_team_size', None)